web/app/components/workflow/nodes/_base/components/field.tsx

```diff
@@ -27,10 +27,10 @@ const Filed: FC<Props> = ({
     toggle: toggleFold,
   }] = useBoolean(true)
   return (
-    <div className={cn(inline && 'flex justify-between items-center', supportFold && 'cursor-pointer')}>
+    <div className={cn(inline && 'flex justify-between items-center')}>
       <div
         onClick={() => supportFold && toggleFold()}
-        className='flex justify-between items-center'>
+        className={cn('flex justify-between items-center', supportFold && 'cursor-pointer')}>
```

Comment:
The `supportFold` panel now can display cursor style correctly.

https://github.com/langgenius/dify/assets/18554747/3ad01c8f-cb6f-4343-9e7b-4d912c033326